wolfssl/wolfio.h

```diff
@@ -163,11 +163,15 @@
             #include <sys/socket.h>
             #include <arpa/inet.h>
             #include <netinet/in.h>
-            #include <netdb.h>
+            #ifdef HAVE_NETDB_H
+                #include <netdb.h>
```

Comment:
Note: This change broke the wolfTPM test, because it (wolfTPM) was not setting `HAVE_NETDB_H`. Fixed in wolfTPM.